docs/sdk/src/guides/mod.rs

```diff
@@ -2,6 +2,22 @@
 //!
 //! This crate contains a collection of guides that are foundational to the developers of
 //! Polkadot SDK. They are common user-journeys that are traversed in the Polkadot ecosystem.
+//!
+//! The main user-journey covered by these guides is:
+//!
+//! * [`your_first_pallet`], where you learn what a FRAME pallet is, and write your first
+//!   application logic.
+//! * [`your_first_runtime`], where you learn how to compile your pallets into a WASM runtime.
+//! * [`your_first_node`], where you learn how to run a node with your runtime.
+//!
+//! > By this step, you have already launched a full blockchain!
```

Comment:
This sounds nice, but the pallets do have `dev_mode`. Could be that people miss that and think they are ready for launch...